holopy/core/io/__init__.py

```diff
@@ -17,5 +17,5 @@
 # along with HoloPy.  If not, see <http://www.gnu.org/licenses/>.
 from holopy.core.io.io import (load, load_image, save, save_image,
     get_example_data, get_example_data_path, load_average)
-from holopy.core.io.vis import (show, test_display, display_image,
-                                show_scatterer_slices)
+from holopy.core.io.vis import show, display_image, show_scatterer_slices
+from holopy.core.io.vis import _test_display as test_display
```

Comment:
Can't we just rename the function in `holopy.core.io.vis` to be `test_display`, instead of calling it `_test_display` and then changing its name on the import here?

Reply:
it was named test_display before this PR, but nose was picking it up and complaining if there was no matplotlib. This PR originally gave it an underscore, but only in the hp.core.io.vis namespace. Docs point to it in the highest-level holopy namespace (under an incorrect name before this PR). 

I also considered having nose skip or disable it, but it's not really a test so I didn't think that was the right solution.

Now I've renamed it back to its original but have it run a separate matplotlib check that warns instead of raising an exception, so nose will run the code but not complain.

Reply:
Ahhh... I see. Can we call it `check_display` then, to avoid nose sniffing it out? Or is it bad form to change this public-facing function on a minor release?

This is a minor issue and an improvement over the old code, so I'm fine with whatever you decide.